app/controllers/admin/radios_controller.rb

```diff
@@ -54,6 +54,17 @@ def set_radio
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def radio_params
-      params.require(:radio).permit(:title, :description, :mp3, :duration, :size, :youtube_url, :podcast_url, :published_at, community_ids: [])
+      params.require(:radio).permit(
+          :title,
+          :description,
+          :mp3,
+          :duration,
+          :size,
+          :youtube_url,
+          :podcast_url,
+          :published_at,
+          community_ids: [],
+          personality_ids: []
```

Comment:
__[RuboCop]__

Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call.


<!-- Comment by [SideCI](https://sideci.com) -->